doc/content/whats-new/3.25.2.md

```diff
@@ -1,6 +1,11 @@
 ---
 date: 2023-05-16
 title: "3.25.2"
+featured: {
+  added: ['Experimental channel steering API, which allows steering end devices from the wide (250kHz or 500kHz) channels towards the narrow (125kHz) channels.', 'This API is mainly relevant for end devices operating in the US915 and AU915 regions, as they may join via a wide channel, but users may want to steer them towards the narrow channels.'],
+  changed: ['Network Operations Center end device uplinks performance retention rate to 30 days.'],
+  fixed: ['In fixed channel plans such as US915 and AU915, the the associated wide (500kHz) channel is now enabled by default.']
+}
```

Comment:
@KrishnaIyer Is this part going to be included in https://github.com/TheThingsIndustries/lorawan-stack-docs/issues/1093, i.e. are `what's new` files going to be autogenerated?

Reply:
Yes good one. This will be a part of #1093

Reply:
@KrishnaIyer With this we should think of a good way to summaries these for example when they contain bullet points, it should be a shorter version

Reply:
Yes indeed. For now we can pay attention to what we add here.